performance/src/efficiency/finding_performance_writer.cpp

```diff
@@ -129,23 +129,33 @@ std::vector<std::vector<measurement>> prepare_data(
  * with its corresponding measurements.
  */
 std::vector<std::vector<measurement>> prepare_data(
-    const track_state_container_types::const_view& track_states_view) {
+    const edm::track_fit_collection<default_algebra>::const_view&
+        track_fit_view,
+    const edm::track_state_collection<default_algebra>::const_view&
+        track_states_view,
+    const measurement_collection_types::const_view& measurements_view) {
     std::vector<std::vector<measurement>> result;
 
-    // Iterate over the tracks.
-    track_state_container_types::const_device track_states(track_states_view);
+    // Set up the input containers.
+    const edm::track_fit_collection<default_algebra>::const_device track_fits(
+        track_fit_view);
+    const edm::track_state_collection<default_algebra>::const_device
+        track_states(track_states_view);
+    const measurement_collection_types::const_device measurements{
+        measurements_view};
```

Comment:
Is there really no nicer way of doing this than tripling the already extreme amount of vecmem boilerplate we have?

Reply:
This is a good find. 🤔 

I wrote this code long ago, before introducing `traccc::edm::track_fit_container`. These could now be replaced by a single `traccc::edm::track_fit_container<traccc::default_algebra>::const_device` type. (Which is to be added to that container type.)